extensions/parquet/base/src/main/java/io/deephaven/parquet/base/materializers/LocalDateTimeMaterializer.java

```diff
@@ -0,0 +1,83 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.parquet.base.materializers;
+
+import io.deephaven.parquet.base.PageMaterializer;
+import io.deephaven.parquet.base.PageMaterializerFactory;
+import io.deephaven.parquet.base.ParquetTimeUtils;
+import org.apache.parquet.column.values.ValuesReader;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.function.LongFunction;
+
+public class LocalDateTimeMaterializer {
+
+    public static final PageMaterializerFactory FromMillisFactory =
```

Comment:
This isn't bad at all. I'm somewhat inclined to suggest following the same pattern as elsewhere, with an abstract fillValues.